### DISCUSSION TITLE
Is it possible to exploit config files to gain access to passwords?

### DISCUSSION THREAD

**Discussion by malyv** [keepassxreboot/keepassxc#9098] [Q&A] [answered]
For example, in keepassxc.ini there is a KeeShare section and a PrivateKey entry. By enabling KeeShare in keepassxc.ini and knowing the PrivateKey can someone share the database and gain access to the passwords?

Or is there some other easy way to change the configuration and get at the passwords without knowing the master password or by tricking the user into providing the master password?

Is a 'secure desktop' functionality planned for entering the master password, similar to the one implemented in KeePass?

**Comment by droidmonkey** [comment, marked as answer]
The answer to the config question is no.

The answer to secure desktop is currently no, but maybe in the future.
